Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -748,13 +763,12 @@
   - ENOUGH CABBAGES
   - FIVE HUNDRED AND NINETY-NINE US DOLLARS
   - FIVE TEENAGERS WITH ATTITUDE
-  - GODDAMN FUCKING PIECE OF SHIT ASSHOLE BITCH-CHRISTING CUNT-SMUGGLING SWEARING
+  - GODDAMN FUCKING PIECE OF SHIT ASSHOLE BITCH-CHRISTING SWEARING #  CUNT-SMUGGLING is ERP since cunt is pussy in slang, this whole thing should actually be removed but need more opinions
```

Comment:
asshole is also erp :trollface:

Reply:
TBH swearing-is-funny is a bit weak anyway

Reply:
asshole is more of an insult in an american setting

Reply:
whole thing removed anyway, since its not that funny and just left over from 13

Reply:
cunt is also an insult in british/australian culture

Reply:
> cunt is also an insult/term of endearment in british/australian culture

yeah im British, cunt commonly means vagina or someone who is a dick but you don't want to say dick because your British and you like sounding British :trollface: